### PR TITLE
Update UG/DG and PPP

### PR DESCRIPTION
**Pull request by AZhiKai**
Further improvements.

**Work in progress**

### CODE REVIEW

**Comment by AZhiKai** [comment]
Hi @dalessr, sorry for the last minute thing, but could you help me review the diagrams for my DG? It's the user account portion.

**Comment by AZhiKai** [comment]
@dalessr thank you so much! :)
Btw for the PE, should we ensure that some data/values are populated when the testers run it? Cos currently when they first launch the app, it only contains a `root` account.

**Comment by dalessr** [comment]
> Btw for the PE, should we ensure that some data/values are populated when the testers run it? Cos currently when they first launch the app, it only contains a root account.

It's good to have some data in the app to make the testing easier for the other teams, but I'm not sure if there is any requirement this sem. Couldn't find it on the website :( Maybe can ask Prof in slack?

**Comment by AZhiKai** [comment]
> > Btw for the PE, should we ensure that some data/values are populated when the testers run it? Cos currently when they first launch the app, it only contains a root account.
> 
> It's good to have some data in the app to make the testing easier for the other teams, but I'm not sure if there is any requirement this sem. Couldn't find it on the website :( Maybe can ask Prof in slack?

@dalessr  Thanks for the reply, I checked the AB4 DG and it seems like we should include some data. So we'll go ahead with that. Thanks again, for your help. :)

File: docs/UserGuide.adoc [hunk 1]
```diff
@@ -106,23 +107,22 @@ e.g. typing *`help`* and pressing kbd:[Enter] will open the help window.
...
+* Requiring users to be authenticated before they can execute any commands.
+* Providing accountability through logging of system events.
 
 === Efficiency
 Time is money. *MeNUS* ensures that the application will:
 
-* Loading within 5 seconds.
-* Executing commands within split of a second and updating the GUI almost instantaneously.
+* Load within 5 seconds.
+* Execute commands within split of a second and update the GUI almost instantaneously.
```
Comment (m4dkip) [inline docs/UserGuide.adoc]: Rephrase to "...within a split second..."

File: docs/UserGuide.adoc [hunk 2]
```diff
@@ -20,36 +20,35 @@ By: `Team CS2103-AY1819S1-F10-4`      Since: `Sep 2018`      Licence: `M
...
+As long as you feel that you are using too many complex and expensive systems to manage your restaurants,
+link:{repoURL}[MeNUS] is here to help you and it is the only restaurant management system you will ever need.
 
 Introducing link:{repoURL}[MeNUS], a revolutionary restaurant management desktop application that is *optimized for
 users who prefer to work with a Command Line Interface (CLI) while still having the benefits of a Graphical User
 Interface (GUI)*. If you can type fast, link:{repoURL}[MeNUS] allows you to manage core aspects of your restaurants --
-employees, menus, inventory, sales and reservations management -- all from a single location!
+accounts (i.e. employees), menus, ingredients, sales and reservations management -- all within a single application!
 
 === Legends
```
Comment (m4dkip) [inline docs/UserGuide.adoc]: Change to "Legend" instead of Legends

File: docs/UserGuide.adoc [hunk 3]
```diff
@@ -20,36 +20,35 @@ By: `Team CS2103-AY1819S1-F10-4`      Since: `Sep 2018`      Licence: `M
...
 The following *3* legends will be used throughout the documentation which you may wish to pay attention to as it may
```
Comment (m4dkip) [inline docs/UserGuide.adoc]: Change "legends" to "symbols"

Because "Legend" refers to the set of symbols

Comment (AZhiKai) [reply docs/UserGuide.adoc]: So Legend or Symbols? :P I feel Symbols isn't intuitive though. I also thought that Legends isn't a good word, but couldn't think of anything else.

**Pull request merged**